### DISCUSSION TITLE
On Stellarium how can I get astrological data from a hundred thousand years Ago?

### DISCUSSION THREAD

**Discussion by tjfoyechplemw** [Stellarium/stellarium#2273] [Q&A]
Hi everyone! My friend is writing a novel and needs lots of astrological data. By using Stellarium he can only get data within one hundred thousand years, and now he needs data beyond this period. Tips from Stellarium directed him to come here for helpe. Please help him, thanks a lot!

**Comment by gzotti** [comment]
What do you need astrological data beyond 100.000 years for? Stellarium can provide astro***nom**ical* planet positions for a few millennia. Accuracy issues are discussed in the User Guide. If you need data beyond sensible limits, you are on your own.

**Reply by tjfoyechplemw** [reply]
Thank you for your answer~
He is not engaged in astronomical research, but the content of the novel he is writing involves some biological evolution research that requires these data. He doesn't need very precise data, just some rough data checksum reference from the star map. He wonders if Stellarium can simulate the starry sky images within 2 million years to him based on the existing data logic, and he is willing to pay.

**Reply by gzotti** [reply]
No, we cannot provide data beyond what we currently have.

**Reply by Arcturios** [reply]
That seems really cool, but it is impossible.
I wouldn't mind going back to 5th Century B.C.E.  years and check out the heavenly bodies at the location of Babylon. 

I wish your friend good luck on the data and the novel.

**Reply by stephanpls** [reply]
You could try your luck with https://celestia.space/
